OPAL/br/src/main/scala/org/opalj/br/fpcf/analyses/immutability/ClassImmutabilityAnalysis.scala

```diff
@@ -118,38 +114,18 @@ class ClassImmutabilityAnalysis(val project: SomeProject) extends FPCFAnalysis {
                 case None =>
                     OPALLogger.warn(
                         "project configuration - class immutability analysis",
-                        s"missing class file of ${t.toJava}; setting all subtypes to mutable"
+                        s"missing class file of ${subtype.toJava}; setting all subtypes to mutable"
                     )
-                    results ::= createResultForAllSubtypes(t, MutableClass)
+                    results ::= createResultForAllSubtypes(subtype, MutableClass)
             }
         }
         IncrementalResult(Results(results), nextComputations.iterator)
     }
 
-    def determineGenericTypeBounds(classFile: ClassFile): Set[(String, String)] = {
```

Comment:
Question: This is just an unused function that somehow ended up as a residual?

Reply:
Looks like it. Don't know why.